iam/policy-templates/dss-lambda.json

```diff
@@ -69,7 +69,6 @@
         "Action": [
             "dynamodb:*"
         ],
-        "Resource": "arn:aws:dynamodb:*:$account_id:table/dss-async-state-$stage",
```

Comment:
Consider making this change in a separate PR to disentangle deploy script changes from IAM policy updates.

Reply:
shall do, ill merge it in first, as it needs to be fixed for the syntax to work in `aws  iam put-role-policy`

Reply:
@xbrianh please see https://github.com/HumanCellAtlas/data-store/pull/2685